src/utils/options.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/python
+
+"""
+    options
+    ~~~~~~~
+
+    Parse all configuration options from the config.ini file for the current site.
+    Additionally, parse the hdw.dat and restrict.dat files of the current site for 
+    other configuration information.
+
+    :copyright: 2023 SuperDARN Canada
+    :author: Theodore Kolkman
+"""
+
+import os
+import json
+from dataclasses import dataclass
+
+
+@dataclass
+class Options:
+    def __post_init__(self):
```

Comment:
I think it would make more sense to declare all the fields of this class before the `__post_init__` method. This way, if someone is curious which members of the Options class they can access, there is very prominently a complete list, rather than having to dig through the `parse_config()` or `parse_hdw()` methods.